libcontainer/setns_init_linux.go

```diff
@@ -34,6 +34,10 @@ func (l *linuxSetnsInit) Init() error {
 	defer runtime.UnlockOSThread()
 
 	if !l.config.Config.NoNewKeyring {
+		if err := label.SetKeyLabel(l.config.ProcessLabel); err != nil {
```

Comment:
Since we are joining an existing keyring here, does it make sense to set a label (if the label is different for a different container, what will happen)?